### DISCUSSION TITLE
Where or how can I visit the OSS list used by GitHub.com ?

### DISCUSSION THREAD

**Discussion by node** [community/OSPO#128] [General]
Is there anyplace that holds the OSS list used by GitHub.com ?   
Thanks.

**Comment by zkoppert** [comment]
Like what GitHub's open source dependencies are? or do you mean something else?

**Reply by node** [reply]
I was just curious to find out what open source software the GitHub site itself uses, so I wanted to see if there was already similar content to refer to, thank you very much for your reply. 🤝

**Comment by ahpook** [comment]
@node for GHES we publish a Credits file that you can get at https://<your url>/site/credits - there is not a direct equivalent for github.com as far as I know.

**Reply by node** [reply]
Thanks for your reply.
Further question is how do I access https:///site/credits ?
